bindings/2.206/GeometryDash.bro

```diff
@@ -1254,16 +1254,16 @@ class CCPartAnimSprite : cocos2d::CCSprite {
 	TodoReturn dirtify();
 	TodoReturn getSpriteForKey(char const*);
 	TodoReturn initWithAnimDesc(char const*, cocos2d::CCTexture2D*, bool);
-	void setColor(cocos2d::ccColor3B);
+	void setColor(cocos2d::ccColor3B*) = win 0x45940;
```

Comment:
:(

Reply:
I assume this is referring to me making it a pointer? Lemme change that back and make a new PR